types/node/fs.d.ts

```diff
@@ -2233,6 +2233,24 @@ declare module 'fs' {
      */
     export function writeSync(fd: number, string: string, position?: number | null, encoding?: BufferEncoding | null): number;
     export type ReadPosition = number | bigint;
+    export interface ReadSyncOptions {
+        /**
+         * @default 0
+         */
+        offset?: number | undefined;
+        /**
+         * @default `length of buffer`
+         */
+        length?: number | undefined;
+        /**
+         * @default null
+         */
+        position?: ReadPosition | null | undefined;
+    }
+    // tslint:disable-next-line:no-unnecessary-generics
```

Comment:
This lint suppression shouldn't be needed, since the generic is actually used.

Reply:
The generic is only used once though, and the linter complains if it's used less than twice. So the options are (a) use the generic a second time, (b) don't use the generic at all, or (c) disable that linting rule.  I don't see a way to do (a) while keeping code concise, but (b) would cause the type-narrowing check to fail, so I went with (c).